site/installing.md

```diff
@@ -240,14 +238,40 @@ Allowable parameters for the launcher URL:
 
 To download and read the Scope manifest run:
 
-    curl --silent -L --remote-name https://cloud.weave.works/launch/k8s/weavescope.yaml
+    curl --silent --remote-name https://cloud.weave.works/k8s/scope.yaml
 
 **Open Scope in Your Browser**
 
-    kubectl port-forward $(kubectl get pod --selector=weave-scope-component=app -o jsonpath='{.items..metadata.name}') 4040
+    kubectl port-forward -n kube-system "$(kubectl get -n kube-system pod --selector=weave-scope-component=app -o jsonpath='{.items..metadata.name}')" 4040
 
 The URL is: http://localhost:4040.
 
+##<a name="ose"></a>Installing Scope on OpenShift
+
+To install Weave Scope on OpenShift, you first need to login as `admin` user:
+
+    oc login -u admin
+
+Next, create a dedicated project for Weave Scope and apply policy permissions to allow Weave Scope to access Kubernetes API:
+
```

Comment:
minor nit:  Next, create a dedicated project for Weave Scope then apply policy permissions that allow...